tests/unit/backend/corpora/dataset_processing/test_process.py

```diff
@@ -239,22 +247,31 @@ def test_extract_metadata_find_raw_layer(self, mock_read_h5ad):
                 "disease",
                 "disease_ontology_term_id",
                 "sex",
+                "sex_ontology_term_id",
                 "ethnicity",
                 "ethnicity_ontology_term_id",
                 "development_stage",
                 "development_stage_ontology_term_id",
+                "organism",
+                "organism_ontology_term_id",
+                "is_primary_data",
+                "cell_type",
+                "cell_type_ontology_term_id",
             ],
             index=(str(i) for i in range(11)),
         )
         uns = {
             "title": "my test dataset",
-            "organism": "Homo sapiens",
-            "organism_ontology_term_id": "NCBITaxon:8505",
-            "layer_descriptions": {"my_awesome_wonky_layer": "raw"},
+            "X_normalization": "normal",
+            "X_approximate_distribution": "normal",
```

Comment:
are we testing the logic for "count" as well?

Reply:
No, but there is no real logic behind `X_approximate_distribution`, it's just a constant value that is passed through as is.

Reply:
Can you point to where this is used? I can try to add some unit tests there.

Reply:
I think there are should be unit tests for the two possible enums https://github.com/chanzuckerberg/single-cell-explorer/blob/main/backend/common/compute/diffexp_generic.py#L33